packages/amplication-server/src/core/build/build.controller.ts

```diff
@@ -44,9 +43,9 @@ export class BuildController {
     EnvironmentVariables.instance.get(Env.CODE_GENERATION_SUCCESS_TOPIC, true)
   )
   async onCodeGenerationSuccess(
-    @Payload() message: KafkaMessage
+    @Payload() message: CodeGenerationSuccess
```

Comment:
@tupe12334 have a look to the amplication nestjs kafka library that contains pipes to standardise this approach 
https://github.com/amplication/amplication/tree/master/libs/util/nestjs/kafka#deserialisation-of-kafka-message